### ISSUE TITLE
sending images from command line

### ISSUE THREAD

**Issue by kw005** [tgalal/yowsup#1144]
Hi,

is there a way to send an image via command line like sending a text message via command line?

Thank you ;)

**Comment by t0rr3sp3dr0** [comment]
No, you will need to use the yowsup-cli to send the image. Or you can write a script using Yowsup library that sends an image parsed as an argument in shell.

**Comment by kw005** [comment]
OK. Thanks for your answer. Does a script like that already exists, are you aware of that?
Thanks you.
